resources/css/home.css

```diff
@@ -1,6 +1,6 @@
 @font-face {
     font-family: "intro-inline";
-    src: url("resources/intro/Intro_Inline.otf");
+    src: url("../intro/Intro_Inline.otf");
 }
```

Comment:
The file for this font was removed. This should be changed to a different font/also removed

Reply:
I realised you might have wanted to delete the readme file, not the font file.

Reply:
Yes soo I closed the issue before maybe wasnt too smart, but this was my response to the fonnt issue:

The client has this font in their Logo, so either they already acquired the proper licences or they are using a different Font. Etiher way this may then be a placeholder for the time being, we would need more details from the client on that one. Since we are not publishing yet, there isnt an issue arrising just now. We need to discuss this with the client then

Reply:
Nevermind, git diff for one of the commits show that it was deleted and I was confused by that. I guess we can leave it for now.